controller/lib/hal/hal.h

```diff
@@ -288,32 +308,58 @@ inline void HalApi::init() {
   watchdog_init();
   constexpr int32_t BAUD_RATE_BPS = 115200;
   Serial.begin(BAUD_RATE_BPS, SERIAL_8N1);
+
+  setDigitalPinMode(PwmPin::BLOWER, PinMode::HAL_OUTPUT);
+  setDigitalPinMode(BinaryPin::SOLENOID, PinMode::HAL_OUTPUT);
 }
 inline uint32_t HalApi::millis() { return ::millis(); }
 inline void HalApi::delay(uint32_t ms) { ::delay(ms); }
+
+inline int HalApi::rawPin(AnalogPin pin) {
+  switch (pin) {
+  case AnalogPin::PATIENT_PRESSURE:
```

Comment:
Is it written down anywhere that we're connecting these sensors to these pins?

Reply:
Pinged Edwin again, https://respiraworks.slack.com/archives/C0100MWE4V8/p1588045739174900?thread_ts=1587936746.167900&cid=C0100MWE4V8

Reply:
The hardware ICD is the controlling doc for parts and the pins they connector to.  This is current for the PCB/STM32 version, but I'll go and add a column for the Breadboard/Arduino version we will be using for the next two-ish weeks until we get the PCB. 
 https://docs.google.com/spreadsheets/d/1JOSQKxkQxXJ6MCMDI9PwUQ6kiuGdujR4D6EJN9u2LWg/edit?usp=sharing

Reply:
Doc has been updated with Arduino-version pinouts

Reply:
doc now also includes the analog scalings

Reply:
Thanks, Edwin.  Comments updated.